maasglobal-schema-generator-ajv/package.json

```diff
@@ -11,7 +11,7 @@
     "mgsg-ajv": "lib/cli.js"
   },
   "dependencies": {
-    "maasglobal-json-schema-validator": "^0.0.6",
+    "maasglobal-json-schema-validator": "^0.0.7",
```

Comment:
I also wonder if adding a deprecated thing to the generator output makes sense. If it is truly deprecated you might want to use it directly in backend without adding code generation support for the deprecated mechanism.